R/quantity-type.R

```diff
@@ -0,0 +1,16 @@
+#' All possible quantity types as defined in OSPSuite

```

Comment:
Can we maybe name all enum files by the pattern "enum-xxx.R"?. Like `enum-quantity-type.R`.

Reply:
I don't think we should name file according to object type. Otherwise some other file should be class-parameter etc..
Why do you propose this change?